src/call.rs

```diff
@@ -100,35 +102,99 @@ impl<'py> PyCallArgs<'py> for () {
 }
 
 impl<'py> PyCallArgs<'py> for Bound<'py, PyTuple> {
+    #[inline]
     fn call(
         self,
         function: Borrowed<'_, 'py, PyAny>,
-        kwargs: Borrowed<'_, '_, PyDict>,
+        kwargs: Borrowed<'_, 'py, PyDict>,
         _: private::Token,
     ) -> PyResult<Bound<'py, PyAny>> {
-        unsafe {
-            ffi::PyObject_Call(function.as_ptr(), self.as_ptr(), kwargs.as_ptr())
-                .assume_owned_or_err(function.py())
-        }
+        self.as_borrowed().call(function, kwargs, private::Token)

```

Comment:
I think I would have passed down the input token in these, instead of constructing a new one. (But I'm fine either way)

Reply:
Agree seems cleaner, had just overlooked 🙈